packages/cli/src/abstract.command.ts

```diff
@@ -104,10 +104,10 @@ export abstract class AbstractCommand {
     const args: ArgsType[] | undefined = Reflect.getOwnMetadata(ARGUMENT_OPTIONS, this.constructor)
 
     console.log(`
-      ${chalk.bold.white(config.name)}\t${config.description}
-      $ ${chalk.green('reapit')} ${parent ? `${chalk.whiteBright(parent.commandOptions.name)} ` : ''}${chalk.white(
-      config.name,
-    )} ${
+    ${parent ? '\t' : ''}${chalk.bold.white(config.name)}\t${config.description}
```

Comment:
This makes my head hurt a bit! Can we do anything to tidy up? Not a big deal though...